doc/hopauth_guide.md

```diff
@@ -74,112 +77,130 @@ Logging in goes through the CILogon service and your institutional single sign o
 
-The management page for a group shows all current members and their level of membership, gives the option to add new group members, lists topics owned by the group, and gives the option to create new topics. 
+To add a member, simply click the 'Add' button for any user in the list at the 'Available Users' section. After adding it to your group, you can select whether that person should be a regular member or also be and owner from the drop-down menu under 'Permission'. Users are listed with both their internal SCiMMA identifier and their email address, which is easier to recognize.
 
-To add a member, simply select the user from the drop-down list in the 'Add Member' section, select whether that person should be a regular member or also be and owner, and then click the 'Add user' button. Users are listed with both their internal SCiMMA identifier and their email address, which is easier to recognize. 
+In the 'Current Members' section of the page any member of the group can be removed or have their membership status switched between regular membership and ownership of the group.
 
-In the 'Current Members' section of the page any member of the group can be removed or have their membership status switched between regular membership and ownership of the group. 
+### Managing Topics
 
-### Creating Topics
+If you are the owner of a group you can create topics owned by your group where you and other members of your group can write (and read) data in Hopskotch. To do so, enter the desired topic name into the field in the 'Create Topic' section of the group topic's management page, and click the 'Create' button. Topic names must be unique across the system, may be 1-249 characters long, and may only contain the characters `a-zA-Z0-9._-`.
 
-If you are the owner of a group you can create topics owned by your group where you and other members of your group can write (and read) data in Hopskotch. To do so, enter the desired topic name into the field in the 'Add Topic' section of the group's management page, and click the 'Create' button. Topic names must be unique across the system, may be 1-249 characters long, and may only contain the characters `a-zA-Z0-9._-`. 
+[![The management page for a topic](hopauth_screenshots/topic_management_g.png)](hopauth_screenshots/topic_management_g.png)
 
-A topic is always accessible to all members of the group which owns it, but by default it is not accessible to any other users. 
+A topic is always accessible to all members of the group which owns it, but by default it is not accessible to any other users.
 
 ### Granting Other Groups Access to Topics
 
-Since, by default, Hopskotch user who are not members of a group cannot access that group's topics, it may be desirable to grant such access in order to share data. 
+Since, by default, Hopskotch user who are not members of a group cannot access that group's topics, it may be desirable to grant such access in order to share data.
```

Comment:
s/Hopskotch user/Hopskotch users/
(This was an old typo, but we might as well fix it now.)